packages/components/src/internal/components/editable/actions.ts

```diff
@@ -1301,7 +1295,7 @@ function parsePastedLookup(column: QueryColumn, descriptors: ValueDescriptor[],
             const vt = v.trim();
             if (vt.length > 0) {
                 const vl = vt.toLowerCase();
-                const vd = descriptors.find(d => d.display && d.display.toString().toLowerCase() === vl);
+                const vd = descriptors?.find(d => d.display && d.display.toString().toLowerCase() === vl);
```

Comment:
This `?` is a code smell to be. This method should not be receiving `undefined`/`null` descriptors.

Reply:
Good point. The repro here was trying to paste a value into the editable grid for the read-only Project/Folder column, which is a lookup. So this code was trying to `parsePastedLookup` for the lookup column even though it will never use the value. I instead made a change in `insertPastedData` so that we check `!readOnlyCol && !readOnlyCell` first.